src/ui/server/helpers/map-cover-period-id/index.ts

```diff
@@ -0,0 +1,16 @@
+import { COVER_PERIOD } from '../../constants';
+
+/**
+ * mapCoverPeriodId
+ * @param {Boolean} answer: cover period answer
+ * @returns {Number} cover period DB ID
+ */
+export const mapCoverPeriodId = (answer: boolean) => {
+  if (answer) {
```

Comment:
Ternary for single line?

Reply:
I think its harder to read as a single 1 line with ~150 characters instead of a few short lines, but have made the change including to the similar `map-total-contract-value` function